processor/src/test/java/org/mapstruct/ap/test/conversion/uuid/UUIDConversionTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.conversion.uuid;
+
+import java.util.UUID;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.runner.AnnotationProcessorTestRunner;
+import org.mapstruct.ap.testutil.runner.GeneratedSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests conversions between {@link java.util.UUID} and String.
+ *
+ * @author Jason Bodnar
+ */
+@RunWith(AnnotationProcessorTestRunner.class)
+public class UUIDConversionTest {
+
+    private final GeneratedSource generatedSource = new GeneratedSource();
```

Comment:
This is not used in this test. You can remove it entirely.

Reply:
Fixed.